packaging/centos7/cloud.spec

```diff
@@ -482,7 +488,7 @@ pip install --upgrade /usr/share/cloudstack-marvin/Marvin-*.tar.gz
 %dir %attr(0770,root,cloud) %{_localstatedir}/cache/%{name}/management/work
 %dir %attr(0770,root,cloud) %{_localstatedir}/cache/%{name}/management/temp
 %dir %attr(0770,root,cloud) %{_localstatedir}/log/%{name}/management
-%config(noreplace) %{_sysconfdir}/sysconfig/%{name}-management
```

Comment:
I am a Debian guy, so I can't give you an authoritative answer. I didn't know that `/etc/default` already existed on CentOS too. I suppose that's up to you. :)